style.css

```diff
@@ -147,17 +148,28 @@ blockquote {
 }
 
 .article-content {
-    width: 320px;
+    width: 100%;
     margin: 15px;
+    padding: 12px;
```

Comment:
that's the biggest problem right now. it must be seamless while now last column has different widths.
![image](https://user-images.githubusercontent.com/104353194/169960977-8d988c6d-3ae6-4ee1-a1cb-03975a065e70.png)

It's my bad, I should have thought about this - I reckon we need to put all this left column in a separate container. Look at this picture - this is how I see it. I tried it and looks nice. Please try 
![image](https://user-images.githubusercontent.com/104353194/169961194-e528ad9e-457d-4bd0-a243-b877f5649ca9.png)

Result: 
![image](https://user-images.githubusercontent.com/104353194/169961359-a2472e41-1e22-4474-b4eb-4994eb855eeb.png)